generator/inceptionmain.go

```diff
@@ -234,6 +241,7 @@ func (im *InceptionMain) Run() error {
 
 		os.Remove(im.TempMainPath)
 		os.Remove(im.exposePath)
+		os.Remove(im.tempDir)
```

Comment:
Probably fine as is with a little more thought, we aren't creating any other files in the dir, so a single `Remove` should work fine.